src/chat.js

```diff
@@ -17,20 +29,57 @@ export function write_chat_msg(
   Object.values(clients).forEach(send_packet)
 }
 
-export default function chat({ server, client }) {
+export default function chat({ server, client, inventory, world }) {
   client.on('chat', (packet) => {
+    if (is_command_function(packet.message)) {
+      log.debug({ sender: client.uuid, command: packet.message }, 'Command')
+
+      return
+    } else {
+      log.debug({ sender: client.uuid, message: packet.message }, 'Message')
+    }
+    const splited_message = packet.message.split(/(%item\d%)/)
+    const formatted_message = splited_message.map((part) => {
```

Comment:
I think we can remove splitted_message and have directly
```suggestion
    const formatted_message = packet.message.split(/(%item\d%)/).map((part) => {
```